seqr/models.py

```diff
@@ -354,6 +349,27 @@ class Meta:
         json_fields = ['last_modified_date', 'created_by']
 
 
+class FamilyNote(ModelWithGUID):
+    NOTE_TYPE_CHOICES = (
+        ('M', 'mme'),
+        ('C', 'case'),
+        ('A', 'analysis'),
+    )
+
+    family = models.ForeignKey(Family, on_delete=models.CASCADE)
```

Comment:
So we could theoretically create a note model mixin that could encapsulate different kinds of notes for different entities, but we would still ultimately need different tables (and thus different models) for different notes because these notes need a foreign key to their family and having a foreign key that could be keyed to different tables is TERRIBLE database design. Since these notes need a "family" relationship, this model/ table can't be used for anything else